public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+
+:marked
+  After inviting heroes, clicking the **Ask** button invokes `announceJob()` that sets the `request` property of `HeroJobBoard` &mdash; and Angular propagates this
+  change to all `HeroPanel` instances:
+
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/send-job-request-ui.png" alt="Job request is sent")
+  
+:marked
+  Now, let's see how a child can communicate with its parent.
+  
+  ## #2: Child to parent communication with event emitting
+  
+  Input property binding is an easy-to-use technique that works great in most parent-to-child communication scenarios. As the expression bound to
+  the input property changes, the modification is immediately propagated to all children. At first we may think that this mechanism could be
+  applied in the opposite direction, to send data from a child to its parent.
+  
+  _Angular does not implement input data binding in this direction._
+  
+  It is a deliberate design decision. Angular was designed with _unidirectional data flow_ in mind. This approach allows better performance 
+  when detecting and propagating property changes, which would not be easily available with a _two-directional data flow_ mechanism. 
+  The unidirectional data flow approach allows avoiding issues coming from recursion, too, when both communicating parties watch for property 
+  changes and at the same time may change the very same property's value.
+  
+  Instead of using a duplex communication channel between parent and child components, we can use two simplex channels, one from the parent to 
+  a child (or children), and another one in the opposite direction, from the child to its parent. While we can use input property binding
+  from parent to child, Angular offers event emitting (output event binding) for child-to-parent communication.
+  
+  ### Establishing output event binding
+  
+  To set up output event emitting, we need to carry out these steps:
+  1. Adding an outpt property to the child component (apply the `@Output()` decorator on the particular property)
+  1. Declaring this property as an `EventEmitter`
+  1. Using this property with the _output event binding syntax_ in the parent template to bind it to an event handler method
+  
+  The Hero Job Board application uses output event binding to sign that a hero undertakes the announced job. The `HeroPanel` component declares 
+  an output property as a type of `EventEmitter`, through which &mdash; as the type name suggests &mdash; it can emit an event. The parent 
+  `HeroJobBoard` component can use the output property to attach an event handler to that particular event.
+  
+  `HeroPanel` declares the `onJobTaken` output event property, as the `@Output` decorator indicates:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'component', 'take-job-event/hero-panel.ts (extract)', {pnk: /(@Output\(\)|takeJob\(\)|onJobTaken\.next)/g})
```

Comment:
**Don't use the *deprecated* `next` method;  use `emit`!**

Find and fix all occurrences.